index.html

```diff
@@ -0,0 +1,1796 @@
+                        </div>
+                        <div class="categorie">
+                          <h3 class="feature-title">
+                            <a href="/">
+                                KBO League Korean Baseball Game Halted After Fire Breaks Out Near Stadium
+                            </a>
+                          </h3>
+                        </div>
+                    </div>
+                    <div class="col-lg-4 opinion">
+                        <div class="invisible block-title font-weight-bold text-muted">SPORTS</div>
+                        <ul class="list-unstyled opinion-title">
+                            <li class="media">
+                              <img class="magic-pic mr-3" src="https://d.newsweek.com/en/full/1587307/michael-jordan-magic-johnson.jpg?w=397&amp;h=265&amp;f=6c7077e8b2f9e3f00026dd70f00b7731" alt="Abdul El-Sayed">
+                              <div class="media-body">
+                                <h6 class="mt-0 mb-1"><a href="/">Magic Johnson Addresses 'The Last Dance' Claims on Isiah Thomas, Dream Team</a></h6>
+                              </div>
+                            </li>
+                            <li class="media">
+                              <img class="aj-pic mr-3" src="https://d.newsweek.com/en/full/1587273/marshawn-lynch-seattle-seahawks.jpg?w=95&amp;h=95&amp;f=ec99e46ae530440018e6bcaace5bca20" alt="...">
+                              <div class="media-body">
+                                <h6 class="mt-0 mb-1"><a href="/">Marshawn Lynch Could Unretire Again and Rejoin the Seattle Seahawks</a></h6>                            
+                              </div>
+                            </li>
+                            <li class="media">
+                              <img class="aj-pic mr-3" src="https://d.newsweek.com/en/full/1587262/wwe-monday-night-raw-aj-styles-returns.jpg?w=95&amp;h=95&amp;l=68&amp;t=25&amp;f=4614454d77ac9ffdd69e32f4f1c575ab" alt="...">
+                              <div class="media-body">
+                                <h6 class="mt-0 mb-1"><a href="/">AJ Styles Returns to WWE 'Monday Night RAW' and Earns MITB Spot</a></h6>
+                              </div>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+                
+                <div class="row magazine-section">
+                    <article class="col-lg-7 col-md-12 col-sm-12">
+                        <div class="text-danger font-weight-bold">SUBSCRIBE</div>
+                        <div class="row no-gutters magazine">
+                            <div class="col-lg-4 col-md-4 col-sm-12">
+                                <img src="https://d.newsweek.com/en/full/1585087/may-08-15-2020-cover.jpg?w=170&amp;q=95&amp;f=c4d10af6474d34e586607b280b6589f0" class="card-img" alt="...">
```

Comment:
This section uses two images in the original webpage. Kindly fix that. I will add a screenshot from the original page.

![image](https://user-images.githubusercontent.com/57414789/81482570-9e7d8100-91fd-11ea-90ed-5b2d4972036f.png)